extension/e2e-tests/addAsset.test.ts

```diff
@@ -8,8 +8,6 @@ test("Adding unverified Soroban token", async ({ page, extensionId }) => {
 
   await page.getByTestId("account-options-dropdown").click();
   await page.getByText("Manage Assets").click({ force: true });
-  await page.getByPlaceholder("Enter password").fill(PASSWORD);
-  await page.getByText("Log In").click({ force: true });
```

Comment:
one byproduct of this ticket is that we are no longer testing the experience when a user's session has timed out. We'll have to think about how we can actually test this in the future 🤔 

Maybe we need to pass a configuration to the build step to make the session timeout immediately instead of wating the normal 24h

Reply:
maybe one way would be to have `sessionTimer.startSession` take the session length as an arg so that in the test we could just pass in a `0`.

Reply:
That's a good call. I think the question is how we want to pass this configuration. Maybe the easiest path would be just appending a query param to `chrome-extension://${extensionId}/index.html` and using that to pass an arg to the background script.

We're going to have a ticket coming up for users to be able to configure how long they want their session to be. This can be added as part of that ticket